src/org/labkey/test/components/ui/grids/ResponsiveGrid.java

```diff
@@ -198,6 +200,14 @@ protected void clickColumnMenuItem(String columnLabel, String menuText, boolean
         waitFor(()-> !menuItem.isDisplayed(), 1000);
     }
 
+    private boolean isElementWithinComponentRectangle(WebElement element)
+    {
+        var containerRect = getComponentElement().getRect();
+        var elementRect = element.getRect();
+        return elementRect.getX() > containerRect.getX();
+    }
+
+
```

Comment:
Is this unused?
```suggestion
```

Reply:
yes, it's unused. this was a crack at testing to see if I could be smart about whether or not the x-scroll is necessary
Abandoned when I couldn't tell if the assumptions (is the component rect what I see, or does it include the scrolled-out-of-view parts?) were true